linode/manager
# feat: [M3-7029] - Add AGLB Certificate Create Drawer

## Description 📝

- Adds Create Certificate Drawer ✨

## Preview 📷
![Screenshot 2023-08-31 at 2 34 38 PM](https://github.com/linode/manager/assets/115251059/ebe3e3a8-43ba-487d-a3cb-8fc10a68ec71)


## How to test 🧪
- Go to `http://localhost:3000/loadbalancers/1/certificates` with the **MSW on** and try to upload a certificate (you can use fake values)
- Nothing will really happen when you submit the form successfully because of the inherent nature of React Query and the MSW
- Check client side validation
